src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -442,6 +443,7 @@ export const buildReverbAnalyticsModel = ({
   statsDestination,
   timePublished,
   timeUpdated,
+  experimentVariant,
```

Comment:
Do we need to add / update any tests for this?

Reply:
Added now 👍